src/components/Paginator/Paginator.selectors.js

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash';
+
+function totalPagesSelector({
+	pageSizeOptions,
+	selectedPageSizeIndex,
+	totalCount,
+	totalPages,
+}) {
+	const pageSize = pageSizeOptions[selectedPageSizeIndex];
+	return _.isNumber(totalPages)
+		? totalPages
+		: totalCount <= pageSize
```

Comment:
For a second I thought this was a reverse fat arrow and I thought you were using some new feature of es6. I was gearing up for a google dive. Then I realized it's just a plain old less than or equal to. 👊